plugins/commands.js

```diff
@@ -34,22 +45,37 @@ exports.plugin = {
             handler: async (request, h) => {
                 const { namespace, name, version } = request.params;
                 const id = `${namespace}-${name}-${version}`;
-
+                let response;
                 let value;
 
-                try {
-                    value = await cache.get(id);
-                } catch (err) {
-                    throw err;
-                }
-
-                if (!value) {
-                    throw boom.notFound();
-                }
+                if (usingS3) {
+                    try {
+                        value = await awsClient.getDownloadStream({ cacheKey: id });
+                        response = h.response(value);
+                        response.headers['content-type'] = 'application/octet-stream';
```

Comment:
is this the current content-type returned when used with catbox-s3 ?

Reply:
yes, return content type is application/octet-stream